sdks/java/harness/src/main/java/org/apache/beam/fn/harness/Caches.java

```diff
@@ -65,8 +65,7 @@ public final class Caches {
    */
   private static final long CACHE_SIZE_CHANGE_LIMIT_BYTES = 1 << 16;
 
-  private static final MemoryMeter MEMORY_METER =
-      MemoryMeter.builder().withGuessing(Guess.BEST).build();
```

Comment:
The API of official jamm 0.4 has adjusted to be different than the community one. BEST is the default for Builder and its not an enum: https://github.com/jbellis/jamm/blob/4b475b906d9a5dcef523db49124fbb8f68e67ee0/src/org/github/jamm/MemoryMeter.java#L589